tests/pkg_u8g2/main.c

```diff
@@ -148,19 +127,33 @@ int main(void)
 
     TEST_DISPLAY(&u8g2, U8G2_R0, u8x8_byte_riotos_hw_spi, u8x8_gpio_and_delay_riotos);
 
-    u8g2_SetPins(&u8g2, pins, pins_enabled);
-    u8g2_SetDevice(&u8g2, SPI_DEV(TEST_SPI));
+    u8x8_riotos_t user_data =
+    {
+        .device_index = TEST_SPI,
+        .pin_cs = TEST_PIN_CS,
+        .pin_dc = TEST_PIN_DC,
+        .pin_reset = TEST_PIN_RESET,
+    };
+
+    u8g2_SetUserPtr(&u8g2, &user_data);
 #endif
 
+
     /* initialize to I2C */
 #if TEST_OUTPUT == TEST_OUTPUT_I2C
     puts("Initializing to I2C.");
 
     TEST_DISPLAY(&u8g2, U8G2_R0, u8x8_byte_riotos_hw_i2c, u8x8_gpio_and_delay_riotos);
 
-    u8g2_SetPins(&u8g2, pins, pins_enabled);
-    u8g2_SetDevice(&u8g2, I2C_DEV(TEST_I2C));
-    u8g2_SetI2CAddress(&u8g2, TEST_ADDR);
```

Comment:
How is the address now passed-on?

Reply:
Thanks for the note - yes, I accidentally deleted the line.